README.md

```diff
@@ -391,7 +391,7 @@ Available controllers:
 - `joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster]`
 
 Notes:
-  - Wrench messages are not displayed properly in Rviz as NaN values are not handled in Rviz and FTS Broadcaster may send NaN values.
+  - Wrench messages are maybe not displayed properly in Rviz as NaN values are not handled in Rviz and FTS Broadcaster may send NaN values.
```

Comment:
```suggestion
  - Wrench messages are may not be displayed properly in Rviz as NaN values are not handled in Rviz and FTS Broadcaster may send NaN values.
```
fixed the english but is this actually true? shouldn't we highlight the recent extension that was made to the rviz plugin?